ci-scripts/ci-runner.bash

```diff
@@ -179,7 +179,7 @@ echo "Running regression on $(hostname) in ${CI_FOLDER}"
 
 if [ $CI_PUBLIC -eq 1 ]; then
     # Run unit tests for the public release
-    swap_files reframe/settings.py reframe/settings.public.py
+    swap_files reframe/settings.py config/generic.py
```

Comment:
Now, I see. We should change a bit the logic here then. You should not swap the files but rather create the correct symlinks. Remember that the default `settings.py` will be a symlink to the `config/generic.py`. So depending on the `CI_PUBLIC` value, you should set accordingly the symlink.